src/api/app/views/webui2/webui/attribute/new.html.haml

```diff
@@ -0,0 +1,20 @@
+- @pagetitle = 'Add Attribute'
+
+.card
+  = render(partial: 'webui/package/tabs', locals: { project: @project, package: @package })
+
+  .card-body
+    %h3= @pagetitle
+    = form_for(@attribute) do |form|
+      .form-row.form-group
+        = form.hidden_field(:project_id)
+        - if @attribute.package_id
+          = form.hidden_field(:package_id)
+        .col-auto
+          = form.collection_select(:attrib_type_id, @attribute_types, :id, :fullname, {}, class: 'form-control')
+        .col.col-form-label
+          - @attribute_types.each do |attribute_type|
+            %p.d-none{ id: "attribute_type-description-#{attribute_type.id}" }
```

Comment:
[nitpick] I find the name of this class `d-none` not very readable :confused:

Reply:
This is part of [Bootstrap](https://getbootstrap.com/docs/4.1/utilities/display/), so I don't have anything to do with the naming...

Reply:
It's a bootstrap class. Live with it ;-)